content/en/docs/tutorials/services/source-ip.md

```diff
@@ -29,7 +29,7 @@ This document makes use of the following terms:
 
 ## Prerequisites
 
-You must have a working Kubernetes 1.5 cluster to run the examples in this
+You must have a working Kubernetes 1.15 cluster to run the examples in this
```

Comment:
This is a very old statement. Rather than changing the version to 1.15, we may want to delete this sentence completely.

Reply:
@tengqm do you mean remove just the version or whole sentence?

Reply:
The whole sentence.

Reply:
@tengqm  IMHO, I don't think we should remove the whole sentence as we need a running cluster to execute examples, maybe we should review and clean up the whole doc.
thoughts?

Reply:
>  review and clean up the whole doc
That does seem like it's needed

Reply:
If anyone's doing a whole-page cleanup, consider using [documentation IP addresses](https://tools.ietf.org/html/rfc5737) in place of the current public IPv4 addresses.
(the RFC1918 private IPv4 addresses are fine to stay as they are).

Reply:
@sftim @tengqm should I create the issue for the same?

Reply:
I think an issue is the right fit here